arlo-client/src/types.ts

```diff
@@ -63,6 +63,7 @@ export interface IAuditBoard {
   id: string
   name: string
   members: IAuditBoardMember[]
+  passphrase?: string
```

Comment:
What's the advantage of marking this as optional? We assume the passphrase is set in `CalculateRiskMeasurement` anyway.

Reply:
@eventualbuddha It is set on the server side, so before we get it back from the `/status` endpoint, it won't exist in the data tree. I expect it would throw errors in `SelectBallotsToAudit` when trying to submit the jurisdictions data with audit boards without passphrases unless they're marked as optional.